flashlib.frc.nt/src/main/java/com/flash3388/frc/nt/NtPidTable.java

```diff
@@ -0,0 +1,82 @@
+package com.flash3388.frc.nt;
+
+import com.flash3388.flashlib.robot.control.PidController;
+import com.flash3388.flashlib.scheduling.actions.Action;
+import com.flash3388.frc.nt.beans.NtDoubleProperty;
+
+import java.util.function.DoubleSupplier;
+
+public class NtPidTable {
+
+    private static final String PID_TABLE_NAME = "Pid";
+    private static final String KP_ENTRY_NAME = "Kp";
+    private static final String KI_ENTRY_NAME = "Ki";
+    private static final String KD_ENTRY_NAME = "Kd";
+    private static final String KF_ENTRY_NAME = "Kf";
+    private static final String PID_INPUT_ENTRY_NAME = "Input";
+    private final NtTable mTable;
+
+    public NtPidTable(double kP, double kI, double kD, double kF, DoubleSupplier processVariableSupplier) {
```

Comment:
What's the point of the supplier if you receive it in `update`?

Reply:
He's cute